### DISCUSSION TITLE
[Question] Couldn't access the app after executing "npm run dev".

### DISCUSSION THREAD

**Discussion by dunmorogher** [vercel/next.js#36093] [Help]
Hello! 

I am new to Next.js and I am following the [tutorial](https://nextjs.org/learn/basics/create-nextjs-app/setup). 
After I did this command `npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn/tree/master/basics/learn-starter"` as there says my app was created well in my local PC (Win10) below.

![圖片](https://user-images.githubusercontent.com/41850822/162952207-136ee33e-0d64-4cba-85ba-2716acc53327.png)

And it seemed all well after I used `npm run dev`. But when I was accessing the app at "http://localhost:3000" the error occurred.

![圖片](https://user-images.githubusercontent.com/41850822/162952958-1610712c-4a1e-4c0a-a794-094b9c4474da.png)

![圖片](https://user-images.githubusercontent.com/41850822/162952796-dce889ab-df37-4b23-82ae-770bc3d891e2.png)

Could you point me out where I did the things wrong? Thanks in advance! :-)

**Comment by icyJoseph** [comment]
Hi, 

I think you need to install `styled-jsx`, not sure why that has turned into a requirement though. For your learning purposes just do:

```shell
npm install styled-jsx
```

For completeness sake, it looks like after `next 12.0.0`, even though `styled-jsx` is added by `next.js`, you still need to manually install it. This does look like a subtle bug. I tested with `11.2.0`, and there the example works without manually installing `styled-jsx`.

@shuding is this perhaps a bug? It happens in both `dev` and `build` modes with anything, and including, above `next 12.0.0`.

**Reply by dunmorogher** [reply]
Hi, thank you @icyJoseph. 

I just did with `npm install styled-jsx` and then it worked... Yesterday I also tried installing "styled-jsx" to fix this issue, but I got an unexpected error so I thought maybe there is anything I did wrong.

**Reply by dunmorogher** [reply]
If I should open one bug for this on issue tab? :-)
